### ISSUE TITLE
Problem install it on Symfony 3.0.6 with composer

### ISSUE THREAD

**Issue by MasterB** [Exercise/HTMLPurifierBundle#38]
It's not possible to install it on Standard Symfony 3.0.6 with composer.
Any plans to do this step?

**Comment by spolischook** [comment]
no, we use this version as well. Please contribute fix, I can create branch and tag new version for symfony 3 support

**Comment by spolischook** [comment]
please make PR to master branch.

**Comment by MasterB** [comment]
Oh, the Symfony 3 fix was committed @christoph-hautzinger on 01.12.15 at [17:42] so it's already done.
The problem is, the new version it not tagged (v0.2.3)

Until then I use:
```
"require": {
        "exercise/htmlpurifier-bundle": "dev-master"
}
```
